ci/builders/linux_android_aot_engine.json

```diff
@@ -235,6 +235,18 @@
                 ],
                 "script": "flutter/testing/symbols/verify_exported.dart",
                 "language": "dart"
+            },
+            {
+                "name": "Generate treemap for android_release_arm64",
```

Comment:
This will require some recipe and config support. Let me think a bit about a generic way to support this use case and I'll get back to you